android/src/main/java/com/urbanairship/reactnative/AirshipModule.kt

```diff
@@ -27,6 +25,8 @@ class AirshipModule internal constructor(val context: ReactApplicationContext) :
     private var isOverrideForegroundDisplayEnabled: Boolean = false
     private val foregroundDisplayRequestMap = mutableMapOf<String, CompletableDeferred<Boolean>>()
 
+    private val scope: CoroutineScope = CoroutineScope(Dispatchers.Main) + SupervisorJob()
```

Comment:
Is there a lifecycle callback we can use to cancel this scope? I think we'd want it to be on destroy of the application (not the hosting Activity)

Reply:
I think modules outlive the activity but i could be wrong. Should we just make it a static var?

Reply:
Hmm I think there is no way to listen for when the application itself is destroyed 🤔